app/app.go

```diff
@@ -233,14 +233,14 @@ func (a *Application) run() {
 			}
 		case newApp := <-a.newAppChan:
```

Comment:
```suggestion
		case newAppInstance := <-a.newAppInstanceChan:
```
These variable names aren't correct any more - we should rename them for clarity